flow-typed/@react-navigation/bottom-tabs_v5.x.x.js

```diff
@@ -540,10 +540,9 @@ declare module '@react-navigation/bottom-tabs' {
   declare type ActionCreators<
     State: NavigationState,
     Action: GenericNavigationAction,
-  > = {
+  > = {|
     +[key: string]: (...args: any) => (Action | State => Action),
-    ...
-  };
+  |};
```

Comment:
Here, there were substantially more *explicitly* inexact object
    types with indexer properties. It doesn't make a lot of sense why
    one would want an inexact object with an indexer property, whether
    implicitly or explicitly inexact. So, make those exact too.

Hmm, yeah.

These are mostly lightly-edited flowgen output, right? Seems like probably a (minor) bug in flowgen.